### PR TITLE
feat: only include func for file or modules

### LINKED ISSUE #28: bug:  Failed to run selected module
# Bug Report

**Starcoin IDE version:**

v0.7.0-alpha

**Current behavior:**

Report error.

image

**Expected behavior:**

Success run module test

**Steps to reproduce:**

```
git clone https://github.com/Elements-Studio/starswap-core.git
install starcoin-ide-v0.7.0-alpha.vsix
test sources/common/SafeMath.move
```

### PR DESCRIPTION
**Pull request by yubing744**
Fix https://github.com/starcoinorg/starcoin-ide/issues/28

### CODE REVIEW

File: src/move-test/run.ts [hunk 1]
```diff
@@ -76,9 +76,21 @@ export class MoveTestRunner {
   // Collect tests
   collecteTests(request: vscode.TestRunRequest, collected: Set<CollectedTest>) {
```
Comment (jiangying000) [inline src/move-test/run.ts:77]: collectTests?

File: src/move-test/run.ts [hunk 2]
```diff
@@ -76,9 +76,21 @@ export class MoveTestRunner {
...
     if (request.include !== undefined) {
-      for (const test of request.include) {
-        collected.add({ item: test, explicitlyIncluded: true });
+      for (const item of request.include) {
+        this.collecteItem(item, collected);
       }
     }
   }
+
+  collecteItem(item: vscode.TestItem, collected: Set<CollectedTest>) {
```
Comment (jiangying000) [inline src/move-test/run.ts:85]: collecteItem -> collectItem?

File: src/move-test/run.ts (hunk 1 above)
Comment (yubing744) [reply src/move-test/run.ts:77]: thanks

File: src/move-test/run.ts (hunk 2 above)
Comment (yubing744) [reply src/move-test/run.ts:85]: thanks

**Comment by yubing744** [comment]
@jiangying000 I submitted a new [PR](https://github.com/starcoinorg/starcoin-ide/pull/37) to fix the problem you mentioned

**Pull request merged**